lib/Runtime/Library/InJavascript/Intl.js

```diff
@@ -81,13 +87,26 @@
     };
 
     var resolveLocaleBestFit = function (locale, defaultLocale) {
-        var resolvedLocale = platform.resolveLocaleBestFit(locale);
+        var resolvedLocale = platform.localeBestFitCache.get(locale);
+        if (resolvedLocale === undefined) {
+            resolvedLocale = platform.resolveLocaleBestFit(locale);
```

Comment:
>resolvedLocale [](start = 12, length = 14)

This can be undefined. Before adding to map should we check that typeof is string?